hbase_sql/src/main/java/com/splicemachine/derby/stream/spark/SparkOperationContext.java

```diff
@@ -197,7 +197,11 @@ public void readExternal(ObjectInput in)
         if(isOp){
             broadcastedActivation = (BroadcastedActivation)in.readObject();
             op=(Op)broadcastedActivation.getActivationHolder().getOperationsMap().get(in.readInt());
-            activation=broadcastedActivation.getActivationHolder().getActivation();
+            ActivationHolder ah = broadcastedActivation.getActivationHolder();
+            activation=ah.getActivation();
+
+            if (ah.needsReinitialization())
+                ah.reinitialize(null, false);
```

Comment:
When we call .reinitialize() on an ActivationHolder we have to call .close(). I haven't inspected the change in depth yet but at first glance it seems we are no closing it. Can you fix that or point me to where we are closing it? Thanks